config/config/etc/spotbugs/exclude.xml

```diff
@@ -83,5 +83,20 @@
         <Method name="parse"/>
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"/>
     </Match>
+    <Match>
+        <!-- Path shadows common/config -->
```

Comment:
what does `Path` mean in these descriptions? I do not understand this explanation